GameLiftPlugin/Scripts~/windows/.export-plugin-zip.ps1

```diff
@@ -0,0 +1,70 @@
+param(
+	[Parameter(Mandatory)]
+	[Alias("ServerSdk", "SdkVersion", "Sdk")]
+	[ValidatePattern("\d+\.\d+\.\d+")]
+	[string] $ServerSdkVersion,
+	[Parameter(Mandatory)]
+	[ValidatePattern("\d+\.\d+\.\d+")]
+	[string] $PluginVersion
+)
+
+$ROOT_DIR=Resolve-Path "$PSScriptRoot\..\..\.."
+$SOURCE_DIR="$ROOT_DIR\GameLiftPlugin\Source"
+$BUILD_DIR="$ROOT_DIR\GameLiftPlugin\build"
+$SERVER_SDK_DIR="GameLift-Cpp-ServerSDK-UnrealPlugin-$ServerSdkVersion";
+$SERVER_SDK_FILENAME="$SERVER_SDK_DIR.zip";
+$DESTINATION_PATH="$SOURCE_DIR\GameliftServer"
+$TEMP_PATH="C:\Temp\AmazonGameLiftPluginUnreal"
+$TEMP_ZIP_PATH="$TEMP_PATH\$SERVER_SDK_FILENAME"
+$TEMP_PLUGIN_PATH="$TEMP_PATH\amazon-gamelift-plugin-unreal"
+$TEMP_EXTRACTED_PATH="$TEMP_PATH\GameLiftServer"
+$ITEMS_TO_COPY_PATH="$TEMP_EXTRACTED_PATH\GameLiftServerSDK\*"
+$SERVER_SDK_S3_LINK_PREFIX="https://gamelift-server-sdk-release.s3.us-west-2.amazonaws.com/unreal"
+$SERVER_SDK_S3_DOWNLOAD_LINK="$SERVER_SDK_S3_LINK_PREFIX/$SERVER_SDK_FILENAME"
+$PACKAGED_ZIP_PATH="$BUILD_DIR\amazon-gamelift-plugin-unreal-$PluginVersion-sdk-$ServerSdkVersion.zip"
+$LISCENSE_FILE_PATH="$TEMP_EXTRACTED_PATH\GameLiftServerSDK\LICENSE_AMAZON_GAMELIFT_SDK.TXT"
+
+if (Test-Path -Path $PACKAGED_ZIP_PATH)
+{
+	Write-Host "Amazon GameLift Server SDK zip has already been prepared, skipping download and extraction." -ForegroundColor DarkYellow
```

Comment:
Please update these comments to specify that this is the Lightweight Unreal Server SDK so there's no confusion with the C++ server sdk